### PR TITLE
fix: add a consistent way to use endpoint path

### LINKED ISSUE #2406: Add configuration option for additional endpoint paths
**Is your feature request related to a problem? Please describe.**
There's currently no way to allow additional paths for endpoints. For example, we want to expose the GraphQL endpoint on both

- /graphql
- /gateway

**Describe the solution you'd like**
A configuration option to allow multiple paths would be nice where you can pass in a list:
```
multiple_paths:
    - /path1
    - /path2
```

### PR DESCRIPTION
**Pull request by bnjjj**
- Fix #2406

**Checklist**

Complete the checklist (and note appropriate exceptions) before a final PR is raised.

- [x] Changes are compatible[^1]
- [x] Documentation[^2] completed
- [ ] Performance impact assessed and acceptable
- Tests added and passing[^3]
    - [x] Unit Tests
    - [ ] Integration Tests
    - [x] Manual Tests

**Exceptions**

*Note any exceptions here*

**Notes**

[^1]. It may be appropriate to bring upcoming changes to the attention of other (impacted) groups. Please endeavour to do this before seeking PR approval. The mechanism for doing this will vary considerably, so use your judgement as to how and when to do this.
[^2]. Configuration is an important part of many changes. Where applicable please try to document configuration examples.
[^3]. Tick whichever testing boxes are applicable. If you are adding Manual Tests:
    - please document the manual testing (extensively) in the Exceptions.
    - please raise a separate issue to automate the test and label it (or ask for it to be labeled) as `manual test`

### CODE REVIEW

File: NEXT_CHANGELOG.md [hunk 1]
```diff
@@ -139,6 +139,22 @@ By [@Geal](https://github.com/geal) in https://github.com/apollographql/router/p
 
 ## 🐛 Fixes
 
+<<<<<<< HEAD
```
Comment (garypen) [inline NEXT_CHANGELOG.md:142]: Need to remove git artifacts.

**Review by garypen — CHANGES_REQUESTED** [review]

File: NEXT_CHANGELOG.md (hunk 1 above)
Comment (bnjjj) [reply NEXT_CHANGELOG.md]: Done. Sorry

File: apollo-router/src/configuration/tests.rs [hunk 2]
```diff
@@ -678,3 +668,46 @@ fn visit_schema(path: &str, schema: &Value, errors: &mut Vec<String>) {
...
+
+#[test]
+fn test_configuration_validate_and_sanitize() {
+    let conf = Configuration::builder()
+        .supergraph(Supergraph::builder().path("/g*").build())
+        .build()
+        .unwrap()
+        .validate()
+        .unwrap();
+    assert_eq!(&conf.supergraph.sanitized_path(), "/g:supergraph_route");
```
Comment (Geal) [inline apollo-router/src/configuration/tests.rs:680]: what is the meaning of the `:supergraph_route` suffix?

Comment (bnjjj) [reply apollo-router/src/configuration/tests.rs:680]: It's just the Axum notation for path parameter

**Review by Geal — APPROVED** [review]

**Review by BrynCooke — APPROVED** [review]

**Pull request merged**